src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1485,22 +1516,40 @@
     <Field Name="Property" Type="PropertySymbol" Null="disallow"/>
     <Field Name="Index" Type="int"/>
   </Node>
+
   <Node Name="BoundDagIndexerEvaluation" Base="BoundDagEvaluation">
     <Field Name="IndexerType" Type="TypeSymbol"/>
     <Field Name="LengthTemp" Type="BoundDagTemp"/>
+
     <!-- Negative Index values indicate positions after a slice pattern -->
     <Field Name="Index" Type="int"/>
-    <Field Name="IndexerAccess" Type="BoundIndexerAccess?"/>
-    <Field Name="IndexerSymbol" Type="PropertySymbol?"/>
-  </Node>
+
+    <!-- May be a plain indexer access or an access to an Index indexer (which may be an implicit indexer) -->
```

Comment:
>plain indexer access or an access to an Index indexer [](http://example.com/codeflow?start=18&length=53)

What is the difference between "plain indexer access" and  "an access to an Index indexer"?

Reply:
Would it be simpler and clearer to enumerate what types of bound nodes can be here?